### ISSUE TITLE
Scheduling during other tasks

### ISSUE THREAD

**Issue by JonPaulMitchell** [rosenpin/Goaly#43]
Tasks are being scheduled concurrently so that I might see something one evening from 7 to 9pm and another Goaly task from 7 to 8pm. I assume the constraint can't be met any other time to satisfy the goal per week but likely a warning that it can't be met is preferable than seeing times when two tasks are scheduled for the same time.

**Comment by rosenpin** [comment]
Is your calendar completely filled out during those times?  
If yes, it might really be due to constraints and we should address it in some way  
But if not, then there is a bug we need to fix with the scheduling

**Comment by JonPaulMitchell** [comment]
No, there is nothing else scheduled outside of Goaly events at those times.
Please let me know what system and or other information you would
appreciate from me to troubleshoot.

**Comment by rosenpin** [comment]
Please go to settings, scroll down and tap send logs to developer.  
We'll take a look at it.  
This sounds like a bug

**Comment by milesdrace** [comment]
I am also getting double scheduled time slots. If I add a new event to my Google calendar, I would hope the goaly algorithm would reassign goals to other open time slots. Schedules change. This is what seemed like the most powerful aspect of the app when I decided to purchase it. Or perhaps it is a bug.

**Comment by rosenpin** [comment]
Fixed in Goaly 2.0. Update the app and tasks won't be scheduled on top of each other anymore.
